### DISCUSSION TITLE
RE2 crash/RE8 low frames

### DISCUSSION THREAD

**Discussion by mikemann137** [praydog/REFramework#527] [General Questions]
I just downloaded the new versions for both of these games and when I launch re2 it runs for about a minute then freezes, sound still plays but nothing works from the headset or the keyboard. 

I got re8 to run but it runs around 10-15 fps. Anyone have any recommendations or advice on to fix either of these?

**Comment by evgn125** [comment]
Have you tried to set the graphic settings to minimum in re8?

**Comment by mikemann137** [comment]
I'm very dumb haha. No I didn't even think to look at it haha.
